doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,174 @@
 [<< Back](../)
 
 # 1. Introduction
+<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
+
+* [1.1 Introduction](#1.1)
+  * [1.1.1 Terminology](#1.1)
+  * [1.1.2 Relation to other communities](#1.1)
+* [1.2 Scope](#1.2)
+* [1.3 Principles and Guidelines](#1.3)
+  * [1.4.1 Overarching Objectives and Goals](#1.4.1)
+* [1.4 Best Practices](#1.4)
+* [1.5 Verification methodologies](#1.5)
+*	[1.6 Assumptions & Dependencies](#1.6)
+*	[1.7 Results Collation & Presentation](#1.7)
+* [1.9 Resources & References](#1.9)
+
+## Executive Summary
+
+The RC2 was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet industry driven quality assurance standards for conformance, verification, and validation. For the purpose of this chapter, infrastructure and cloud native network function (CNF) testing will be performed to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of software delivery.  More specifically, Conformance includes:
+
+ - Verified implementations of infrastructure and CNF match expected design requirements
+ - Clearly stated guidelines for test, badging, and lifecycle management
+ - Inclusion of Operational run-books for 3rd party supplier instantiation and validations
+ - Evidence, through test results, confirming delivered code matches industrial requirements
+ - Interoperability testing with Reference CNFs, ensuring integration stability and life-cycle management of the Reference CNF on the target implementation.
+
+In summary, **Conformance** testing will be performed for **Verification** and **Validation** purposes, defined further as:
+
+- **Verification** will be used to indicate conformance to design requirement specifications. Accomplished with Requirement Traceability and Manifest Reviews to ensure the Kubernetes infrastructure is delivered per implementation specifications.
+- **Validations** is used to indicate that testing performed confirms the infrastructure and CNF meet the expected, or desired outcome, or behaviour.
```

Comment:
```suggestion
- **Validation** is performed using testing to confirm that the cloud infrastructure and CNF meet the expected, desired outcomes, and behaviours.
```

Reply:
can you add "or" before desired then I will merge it

Reply:
@xmulligan Done. Thanks